scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# This file is part of qutebrowser.
+
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+'''Tool to import browser history data from other browsers. Although, safari
+support is still on the way.'''
```

Comment:
Please use double-quotes for docstrings, and have a summary line on the top (see [pep257](https://www.python.org/dev/peps/pep-0257/)). Something like:

```python
"""Tool to import browser history data from other browsers.

Safari support is still on the way."""
```

OTOH, I think we spoke about this in IRC and decided that it's probably not worth supporting Safari? Then this could probably be dropped so it's just a one-line docstring.